src/ui/triggers_main_area.ui

```diff
@@ -647,33 +644,47 @@
          <property name="bottomMargin">
           <number>3</number>
          </property>
-         <item>
+         <item row="0" column="0">
+          <widget class="QLabel" name="label_foregroundColor">
+           <property name="text">
+            <string>Foreground</string>
+           </property>
+           <property name="buddy">
+            <cstring>pushButtonFgColor</cstring>
+           </property>
+          </widget>
+         </item>
+         <item row="0" column="1">
+          <widget class="QLabel" name="label_backgroundColor">
+           <property name="text">
+            <string>Background</string>
+           </property>
+           <property name="buddy">
+            <cstring>pushButtonBgColor</cstring>
+           </property>
+          </widget>
+         </item>
+         <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonFgColor">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
-           <property name="text">
-            <string>Foreground</string>
+           <property name="flat">
```

Comment:
Actually this property is not needed here - the buttons are not flat by default - so this is just noise (it might be possible to remove this within the Qt Designer plug in by clicking the small clear/revert icon to the extreme right of the checkbox that controls this property so that the item is no longer bold) as it is it means that this button has this property (not set) but the other one does not state anything for this property. From an OCD point of view this is not perfect.

Bold - so specified even if not set away from default:
![Screenshot_20200429_032929](https://user-images.githubusercontent.com/6163092/80555947-d4369480-89c9-11ea-843f-cbbbab0dcb8c.png)

Reset to default button is active on item:
![Screenshot_20200429_032953](https://user-images.githubusercontent.com/6163092/80555984-f6301700-89c9-11ea-9a9e-7890a76fdb3a.png)

Clicked to reset item - note that the checkbox state of the item has not changed:
![Screenshot_20200429_033013](https://user-images.githubusercontent.com/6163092/80556022-18299980-89ca-11ea-8a4e-1f2e530e8c55.png)

But also note that the item is now not bold so indicates that it is has not been set/reset away from the default (and I expect it NOT to be present in the XML):
![Screenshot_20200429_033029](https://user-images.githubusercontent.com/6163092/80556105-69398d80-89ca-11ea-9f74-5bebd6ba930c.png)

And indeed the version of the button that I was playing with (in a different branch) does not show any details for the widget item's flat property:
![image](https://user-images.githubusercontent.com/6163092/80556232-d9481380-89ca-11ea-98fb-053e1e85261f.png)